cephadm-preflight.yml

```
@@ -48,6 +48,205 @@
     - name: redhat family of OS related tasks
       when: ansible_facts['os_family'] == 'RedHat'
       block:
+        - name: Initialize os_check variable
+          set_fact:
+            os_check: { "failed": false }
```

I don't get why you need this fact, it seems not needed at all.
```suggestion
          set_fact:
            os_check: { "failed": false }
```